com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/LitSurfaceInputsUIBlock.cs

```diff
@@ -483,20 +461,6 @@ void DrawSurfaceInputsGUI()
                     }
                     EditorGUI.indentLevel--;
                 }
-
-                // UI only updates intermediate values, this will update the values actually used by the shader.
-                if (EditorGUI.EndChangeCheck())
-                {
-                    SurfaceOptionUIBlock surfaceOption;
-
-                    // Fetch the surface option block which contains the function to update the displacement datas
-                    if (m_LayerCount == 1)
```

Comment:
sanity check, why is it removed? or this part is move to runtime (haven't check all the code yet)